doc/nnn.txt

```diff
@@ -96,9 +96,10 @@ Defaults mappings. Can be disabled by setting |g:nnn#set_default_mappings| to
 
```

Comment:
Here it says that the binding can be disabled via `g:nnn#set_default_mappings` but that's not true with how `<C-w> is implemented currently.